src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -111,14 +126,87 @@
             <form:checkbox path="showNotesToStudents" id="showNotesToStudents"/> Show Notes entered on Class Roster page to students
         </label>
         <br/>
+            <label>
+                <form:checkbox  path ="gradingOn" id="conversionConfirm"/> Convert Attendance to Assignment
+            </label>
+        <br/>
+
+        <div class = "container-fluid ${courseConfigurationForm.gradingOn? '' : 'hidden'}" id="conversionConfig" >
+            <br/>
+            <label> NOTE: When this assignment is pushed to the gradebook, it will immediately be published. Please do not alter the assignment in the gradebook, but instead use this application to update the assignment as needed.
+            </label>
+            <br/>
+            <div class="col-md-2 col-md-offset-0">
+                <label for="assignmentName">
+                    <h5><i>Assignment Name: </i></h5>
+                    <form:input type = "text" path ="assignmentName" id = "assignmentName" size = "15"/>
+                </label>
+                <br/>
+                <label for="assignmentPoints">
+                    <h5><i>Total Points: </i></h5>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)